lib/vdc/ConstantGrid.cpp

```diff
@@ -39,14 +39,14 @@ std::vector<size_t> ConstantGrid::GetCoordDimensions(size_t) const
 
 size_t ConstantGrid::GetGeometryDim() const { return 3; }
 
-const std::vector<size_t> &ConstantGrid::GetNodeDimensions() const
+const std::array<size_t, 3> ConstantGrid::GetNodeDimensions() const
```

Comment:
Please use Size_tArr3 for the type.

Reply:
Well, this alias is so confusing---it *appears* to be using both camel case and underscore conventions simultaneously within 10 letters. How about we use the C++ standard name for this PR and wait the alias to be fixed to be less confusing later?

Reply:
@shaomeng I'd like the code to be consistent. We have a developer-defined type for all of these 3-element array arguments, please use it. If you really have a problem with the naming of the type you are more than welcome to change it.

Reply:
Ok, I've renamed the alias in 52081bac1 with two of them: `DimsType` and `CoordType`. Two notes:
1. It's common practice to have a word "type" at the end of all alias, so when people see it people know it's an alias that redefines something else.
2. It's also common to name alias after its semantic meaning, (Dims and Coord in this example), so the following function signature is very easy to understand:
```c++
void function( DimsType, CoordType );

Reply:
That's fine by me. FYI it is also common practice to designate types with "_t" suffix. E.g. size_t, uint_t, etc.